[2025-04-05T19:43:54.010+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-05T19:43:54.060+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: iniciar_tabla_mysql.start_table manual__2025-04-05T19:43:51.711801+00:00 [queued]>
[2025-04-05T19:43:54.077+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: iniciar_tabla_mysql.start_table manual__2025-04-05T19:43:51.711801+00:00 [queued]>
[2025-04-05T19:43:54.078+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-04-05T19:43:54.113+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): start_table> on 2025-04-05 19:43:51.711801+00:00
[2025-04-05T19:43:54.134+0000] {standard_task_runner.py:63} INFO - Started process 89730 to run task
[2025-04-05T19:43:54.157+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'iniciar_tabla_mysql', 'start_table', 'manual__2025-04-05T19:43:51.711801+00:00', '--job-id', '655', '--raw', '--subdir', 'DAGS_FOLDER/mysql_start_table.py', '--cfg-path', '/tmp/tmpoanth7s7']
[2025-04-05T19:43:54.170+0000] {standard_task_runner.py:91} INFO - Job 655: Subtask start_table
[2025-04-05T19:43:54.260+0000] {task_command.py:426} INFO - Running <TaskInstance: iniciar_tabla_mysql.start_table manual__2025-04-05T19:43:51.711801+00:00 [running]> on host 1e5dd8b23497
[2025-04-05T19:43:54.441+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='iniciar_tabla_mysql' AIRFLOW_CTX_TASK_ID='start_table' AIRFLOW_CTX_EXECUTION_DATE='2025-04-05T19:43:51.711801+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-05T19:43:51.711801+00:00'
[2025-04-05T19:43:54.443+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-05T19:43:54.487+0000] {base.py:84} INFO - Using connection ID 'mysql_***_conn' for task execution.
[2025-04-05T19:43:54.642+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-05T19:43:54.644+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-05T19:43:54.659+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=iniciar_tabla_mysql, task_id=start_table, execution_date=20250405T194351, start_date=20250405T194354, end_date=20250405T194354
[2025-04-05T19:43:54.720+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-05T19:43:54.753+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-05T19:43:54.754+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
